security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -15,6 +15,39 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
 
 To access the latest release, see xref:../../security/compliance_operator/co-management/compliance-operator-updating.adoc#olm-preparing-upgrade_compliance-operator-updating[Updating the Compliance Operator].
 
+[id="compliance-operator-release-notes-1-4-1"]
+== OpenShift Compliance Operator 1.4.1
+
+The following advisory is available for the OpenShift Compliance Operator 1.4.1:
+
+* link:https://access.redhat.com/errata/RHBA-2024:1830[RHBA-2024:1830 - OpenShift Compliance Operator bug fix and enhancement update]
+
+[id="compliance-operator-1-4-1-new-features-and-enhancements"]
+=== New features and enhancements
+
+* With this update, Compliance Operator now provides `OCP4 STIG ID` and `SRG` with the profile rules. (link:https://issues.redhat.com/browse/CMP-2401[*CMP-2401*])
+
+* With this update, obsolete rules being applied to `s390x` have been removed. (link:https://issues.redhat.com/browse/CMP-2471[*CMP-2471*])
+
+[id="compliance-operator-1-4-1-bug-fixes"]
+=== Bug fixes
+
+* Previously, for RHCOS systems using RHEL9, application of the `upstream-ocp4-kubelet-enable-protect-kernel-sysctl-file-exist` rule would fail. With this update, after auto remediation is applied, RHEL9-based RHCOS systems will show `PASS` upon the application of this rule. (link:https://issues.redhat.com/browse/OCPBUGS-13589[*OCPBUGS-13589*])
+
+* Previously, after applying compliance remediations using profile `rhcos4-e8`, the nodes are no longer accessible via SSH using the core user account. With this update, nodes remain accessible via SSH using sshkey. (link:https://issues.redhat.com/browse/OCPBUGS-1833[*OCPBUGS-1833*])
```

Comment:
You did a very good job of using the same structure in the Bug Fixes section, and I have a small suggestion for your to consider. I think the line describing the previous behavior might read just a little more clearly if it makes use of past tense. For example instead of:

"Previously, after applying compliance remediations using profile `rhcos4-e8`, the nodes are no longer accessible via SSH using the core user account. With this update, nodes remain accessible via SSH using sshkey."

How about something like this:

Previously, after applying compliance remediations using profile `rhcos4-e8`, the nodes were no longer accessible via SSH using the core user account. With this update, nodes remain accessible via SSH using sshkey.

Reply:
Agreed